src/scheduler/node_info.c

```diff
@@ -6307,6 +6304,9 @@ check_node_eligibility_chunk(th_data_nd_eligible *data)
 						for (j = 0; node->hostset->ninfo_arr[j] != NULL; j++) {
 							node_info *n = node->hostset->ninfo_arr[j];
 							n->nscr.ineligible = 1;
+							set_schd_error_codes(misc_err, NOT_RUN, NODE_NOT_EXCL);
+							translate_fail_code(misc_err, NULL, exclerr_buf);
+							clear_schd_error(misc_err);
```

Comment:
Just make a call to schdlogerr() instead of doing the translate yourself.  It will not translate if we're not going to print it.